instances/technique/electrocardiography.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/electrocardiography",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "Electrocardiography is the process of producing an electrogram of the heart: a graph of voltage versus time of the electrical activity of the heart using electrodes placed on the skin.",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0103701",
+  "knowledgeSpaceLink": null,
+  "name": "electrocardiography",
+  "preferredOntologyIdentifier": null,
+  "synonym": null
```

Comment:
I would add the following synonym: "ECG"

Reply:
Done.